### DISCUSSION TITLE
Porting Moria possible?

### DISCUSSION THREAD

**Discussion by adaumann** [drmortalwombat/oscar64#176] [Q&A]
Hi, 
I had the crazy idea to port the game Moria (and extend it graphically) from the orginal sources utilizing Easy Flash and the automatic bank switching template (I have now 21 banks!). Now it compiles after a few changes, but the heap and stack does not fit. I played around with the memory settings but no luck. Most likely it will never work on a c64, but maybe you see a change. I can share the repo if you like... I think it will be a great thing to have a classic Unix RL on the C64... 

Andy

**Comment by drmortalwombat** [comment]
I will have a look at the memory footprint if you like.

**Reply by adaumann** [reply]
I think it is pointless, there is far to much data... maybe I do the same with original rogue. It should be less data.
